required/latex-lab/latex-lab-namespace.dtx

```diff
@@ -203,17 +207,28 @@ tables,       Sect,     pdf2,
 %<@@=tag>
 %    \end{macrocode}
 %
+% 
 % \begin{macro}{\AssignStructureRole}
-% We redefine \cs{AssignStructureRole} to setup that tagging rolemapping
+%    We redefine \cs{AssignStructureRole} to setup that tagging
+%    rolemapping.  The second argument needs expansion so that we do
```

Comment:
Not sure what the first sentence is saying

Reply:
Made a changer but not sure either what you meant here @u-fischer.